tests/test_pandas_api.py

```diff
@@ -2029,3 +2029,154 @@ def test_keyed_loc_fixes(q):
         mkt[['k1', 'y']]
     with pytest.raises(KeyError):
         mkt['k1']
+
+
+def test_pandas_count(q):
+    tab = q('([] k1: 0n 2 0n 2 0n ; k2: (`a;`;`b;`;`c))')
+    df = tab.pd()
+
+    qcount = tab.count(axis=1).py()
+    pcount = df.count(axis=1)
+    
+    assert int(qcount[0]) == int(pcount[0])
+    assert int(qcount[1]) == 1
+
+    qcount = tab.count().py()
+    pcount = df.count()
+
+    assert int(qcount["k1"]) == int(pcount["k1"])
+    assert int(qcount["k2"]) == 3
+
+    qcount = tab.count(numeric_only=True).py()
+    pcount = df.count(numeric_only=True)
+
+    assert int(qcount["k1"]) == int(pcount["k1"])
+
+
+def test_df_add_prefix(kx, q):
+    t = q('([] til 5; 5?5; 5?1f; (5;5)#100?" ")')
+
+    q_add_prefix = t.add_prefix("col_", axis=1)
+
+    assert(q('~', q_add_prefix, t.pd().add_prefix("col_", axis=1)))
+
+    kt = kx.q('([idx:til 5] til 5; 5?5; 5?1f; (5;5)#100?" ")')
+
+    q_add_prefix = kt.add_prefix("col_", axis=1)
+    assert(q('~', q_add_prefix, kt.pd().add_prefix("col_", axis=1)))
+
+    with pytest.raises(ValueError):
+        t.add_prefix("col_", axis=0)
+    
+    with pytest.raises(ValueError):
+        t.add_suffix("col_", axis=3)
+
+def test_df_add_suffix(kx, q):
+    t = q('([] til 5; 5?5; 5?1f; (5;5)#100?" ")')
+
+    q_add_suffix = t.add_suffix("_col", axis=1)
+
+    assert(q('~', q_add_suffix, t.pd().add_suffix("_col", axis=1)))
+
+    kt = kx.q('([idx:til 5] til 5; 5?5; 5?1f; (5;5)#100?" ")')
+
+    q_add_suffix = kt.add_suffix("_col", axis=1)
+    assert(q('~', q_add_suffix, kt.pd().add_suffix("_col", axis=1)))
+
+    with pytest.raises(ValueError):
+        t.add_suffix("_col", axis=0)
+    
+    with pytest.raises(ValueError):
+        t.add_suffix("_col", axis=3)

```

Comment:
It would be good for these tests to additionally be testing the output is as expected, so for example in this case the following would be useful, it will limit the chances of silent error regressions going forward and something we've been more conscious of recently

```suggestion
    with pytest.raises(ValueError) as err:
        t.add_suffix("_col", axis=3)
    assert 'No axis named 3' in str(err)
```

Reply:
Hi Connor, thanks for your advice. We have already included it.